_drafts/oryx/technology.md

```diff
@@ -0,0 +1,63 @@
+# Oryx technologies
+
+Oryx is a frontend framework that is designed to help developers build web applications using modern technologies.
+When possible and appropriate, Oryx adheres to standards for both technologies and tools.
+Oryx strives to minimize the use of build tools and instead utilize the native capabilities of the web, as much as possible.
+Oryx aims to use JavaScript during development, but tries avoid the need to run JavaScript in the browser. Wherever possible, HTML and CSS are preferred to render a fast loading experience cross different screen sizes.
+
+We will examine both the deployed technology stack and the technologies utilized for development in order to gain a more comprehensive understanding of the technologies.
+
+## Deployed stack:
+
+### HTML & CSS & JS
+HTML, CSS, and JS are considered to be the fundamental and essential set of web technologies used for creating web pages. HTML is responsible for the structure and content of the page, CSS is used for styling and layout, and JS adds interactivity and dynamic behavior to the page. Together, these three technologies form the foundation of modern web development.
+
+### Web Components
+Web Components are a set of web platform APIs that allow developers to create custom, reusable, and encapsulated HTML elements.
+
+### Web hosting
+Oryx uses Netlify as a hosting provider, which means you can easily find basic boilerplate to deploy your site on Netlify. However, if you choose to use a different hosting provider, there may be limitations on the implementation of SSR. To make it work, you would need to adjust the SSR rendering boilerplate, which is specific to the hosting provider.
```

Comment:
Maybe rephrase it to something like "currently" or "by default"? As it sounds a bit like netlify is our preffered or final choice. @tobi-or-not-tobi ?

Reply:
Good point, added that we are currently using it.

Reply:
We c/should actually drop the notion of Netlify entirely. There's no dependency on Netlify, but we do have a toml file and some integration with lambda which might make it opinionated